### PR TITLE
[2.0] Add script used to deploy blobs to Azure storage

### PR DESCRIPTION
**Pull request by natemcmaster**
Let's stop doing this by hand.

TODO:
How can we reliably also fetch the assets for the preview 2.0 build? At the moment, I've been manually copying them. Thoughts @JunTaoLuo ?

### CODE REVIEW

**Comment by natemcmaster** [comment]
Talked in person with @JunTaoLuo. Let's not both automating manually copying previous releases. Instead, let's cleanup the way we produce drop shares.

**Comment by natemcmaster** [comment]
FYI @muratg - merging as this is something we would have done by hand anyways. It doesn't impact the build yet as the automation to run this script isn't wired up.

File: scripts/UploadBlobs.ps1 [hunk 1]
```diff
@@ -0,0 +1,142 @@
+<#
```
Comment (JunTaoLuo) [inline scripts/UploadBlobs.ps1]: Where does this copy over the artifacts from previous release?

Comment (natemcmaster) [reply scripts/UploadBlobs.ps1]: It doesn't. I'm still doing that manually.

**Pull request merged**